README.md

```diff
@@ -1,2 +1,18 @@
 # zm-migration-tool
 Zimbra migration tool
+
+Ant command to create jar
+
+ant zmzimbratozimbramig
+
+This will create jar in location build/jars/zmzimbratozimbramig.jar
+![image](https://user-images.githubusercontent.com/37224867/170489962-e70352f2-06ad-4e09-8d80-bd288b25a1d3.png)
+
+
+Build information :-
+
+Any change in this repo needs an version update in following places.
+1.  zmzimbratozimbramig.version in build.xml
+2. zm-zcs-lib in build.xml
```

Comment:
I didn't get this, Do we have to update the version in the file or do we have to change jar version? 
Confused because did not see anything in build.xml for zm-zcs-lib

Reply:
Once we create the jar we upload it to files.zimbra.com with updated version and then upgrade the version in zm-zcs-lib before we merge zm-zcs-lib in develop.